iosApp/iosApp/ui/pages/LandingScreen.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftUI
+import Shared
+import KMMViewModelSwiftUI
+
+
+
+struct LandingScreenView: View {
+    var body: some View {
+        ZStack {
+            Image("landingbackground")
+                .resizable()
+                .scaledToFill()
+                .edgesIgnoringSafeArea(.all)
+            
+            VStack(alignment: .leading) {
+                HeaderComponent()
+                Spacer()
+                ButtonsComponent()
+            }.padding(.horizontal, 16)
+                .padding(.top, 46)
+        }
+    }
+}
+
+struct HeaderComponent: View {
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Text("Stroomlijn je dag met")
```

Comment:
Is this hardcoded text or how does this work?

Reply:
Yea this is hardcoded text for now. Is it fine to keep this for now, since we're going to use shared resources whenever all PRs are merged?